### DISCUSSION TITLE
Deleting old sensor data after SO2 upgrade

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#3567] [Unsupported Versions] [answered]
Can the `/nsm/sensor_data` directory be deleted after completing all the steps the SO1 -> SO2 [migration instructions](https://docs.securityonion.net/en/2.3/appendix.html)?

**Comment by weslambert** [comment, marked as answer]
Yes, there should be no issue with deleting that particular directory (providing you don't actually delete `/nsm` itself 😄  )

**Reply by kiaraho** [reply]
😆
